<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>694</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>694</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>694</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Harley Davidson Trouble Code</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/logos/window_logo.png</normaloff>:/logos/window_logo.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-1</y>
      <width>801</width>
      <height>291</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="HD_LOGO">
     <property name="geometry">
      <rect>
       <x>-4</x>
       <y>2</y>
       <width>441</width>
       <height>291</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-image: url(:/logos/hd_logo.png);
background-color: rgb(0, 0, 0);
background-repeat:no-repeat;
background-position:center;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="code_input">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>120</y>
       <width>301</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(50,50,50);
border-radius:10px;
color: rgb(201,52,19);</string>
     </property>
     <property name="maxLength">
      <number>10</number>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <property name="cursorPosition">
      <number>0</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>532</x>
       <y>62</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(201,52,19);</string>
     </property>
     <property name="text">
      <string>ENTER CODE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="submit_code">
     <property name="geometry">
      <rect>
       <x>528</x>
       <y>210</y>
       <width>171</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-color: rgb(35,35,35);
	border:2px solid rgb(60,60,60);
	border-radius:10px;
	color: rgb(201,52,19);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
}
QPushButton:pressed {	
	background-color: rgb(201,52,19);
	border:2px solid rgb(201,52,19);
	color: rgb(200,200,200);
}</string>
     </property>
     <property name="text">
      <string>SUBMIT</string>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
    <widget class="QPushButton" name="about_btn">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>10</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-color: rgb(35,35,35);
	border:2px solid rgb(60,60,60);
	border-radius:10px;
	color: rgb(201,52,19);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
}
QPushButton:pressed {	
	background-color: rgb(201,52,19);
	border:2px solid rgb(201,52,19);
	color: rgb(200,200,200);
}</string>
     </property>
     <property name="text">
      <string>ABOUT</string>
     </property>
    </widget>
    <widget class="QPushButton" name="edit_motorcycles_button">
     <property name="geometry">
      <rect>
       <x>574</x>
       <y>10</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-color: rgb(35,35,35);
	border:2px solid rgb(60,60,60);
	border-radius:10px;
	color: rgb(201,52,19);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
}
QPushButton:pressed {	
	background-color: rgb(201,52,19);
	border:2px solid rgb(201,52,19);
	color: rgb(200,200,200);
}</string>
     </property>
     <property name="text">
      <string>EDIT MOTORCYCLES</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>69</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(35,35,35);
color: red;
border-radius: 10px;</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>288</y>
      <width>801</width>
      <height>411</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="error_output">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>10</y>
       <width>521</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(50,50,50);
border-radius:10px;
color: rgb(201,52,19);</string>
     </property>
     <property name="lineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(201,52,19);</string>
     </property>
     <property name="text">
      <string>ERROR OUTPUT</string>
     </property>
    </widget>
    <widget class="QLabel" name="system_output">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>95</y>
       <width>521</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(50,50,50);
border-radius:10px;
color: rgb(201,52,19);</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>115</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>16</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(201,52,19);</string>
     </property>
     <property name="text">
      <string>SYSTEM</string>
     </property>
    </widget>
    <widget class="QListWidget" name="log_area">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>181</y>
       <width>591</width>
       <height>211</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(195,195,195);
border-radius:10px;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save_log">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>320</y>
       <width>171</width>
       <height>71</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {	
	background-color: rgb(35,35,35);
	border:2px solid rgb(60,60,60);
	border-radius:10px;
	color: rgb(201,52,19);
}
QPushButton:hover {	
	background-color: rgb(60,60,60);
	border:2px solid rgb(70,70,70);
}
QPushButton:pressed {	
	background-color: rgb(201,52,19);
	border:2px solid rgb(201,52,19);
	color: rgb(200,200,200);
}</string>
     </property>
     <property name="text">
      <string>SAVE</string>
     </property>
     <property name="shortcut">
      <string>Return</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
